databroker/experimental/schemas.py

```diff
@@ -18,15 +19,15 @@
 }
 
 
-class Document(pydantic.BaseModel):
+class BaseDocument(pydantic.BaseModel):
     key: str
     structure_family: StructureFamily
     structure: Union[ArrayStructure, DataFrameStructure]
     metadata: Dict
     specs: List[str]
     mimetype: str
-    data_blob: Optional[bytes]
-    data_url: Optional[pydantic.AnyUrl]
+    created_at: datetime
+    updated_at: datetime
```

Comment:
Do you want to move this to just `Document`? You argued that we should only write it down in one place, and I'm convinced.

Reply:
For this particular case, I think that we could move `created_at` to `Document` and leave `updated_at` in `BaseDocument`. This way, we can keep track of when a document version was created before we move it to revisions

Reply:
Oops, yes, I agree. I meant to aim this comment at `created_at` but I hit the wrong line. The current version and the past revisions should each have `updated_at` but only the current version needs `created_at`.